packages/get-npm-dependencies-for-file/README.md

```diff
@@ -0,0 +1,19 @@
+# get-npm-dependencies-for-file
+
+Analyze a file's requires/imports with [precinct](https://github.com/dependents/node-precinct) and returns an array of NPM dependencies.
+
+Typescript, ES6, and commonjs modules are all supported.
+
+## Install
+
+`npm i get-npm-dependencies-for-file`
+
+## Usage
+
+```js
+const path = require(`path`)
```

Comment:
Add a new util package `get-npm-dependencies-for-file` for finding the dependencies for each function.